src/main/java/mate/academy/Main.java

```diff
@@ -6,16 +6,20 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
```

Comment:
Ensure that the logger is configured with appenders for both File and Console as per the task requirements .